.github/workflows/e2e-test.yml

```diff
@@ -1,11 +1,8 @@
-name: WCPay E2E Tests
+name: E2E Tests - Full
```

Comment:
`All` might be a better terminology here?

Reply:
Updated in https://github.com/Automattic/woocommerce-payments/pull/4658/commits/7a37e169fba2da2f25acf414675a681e081b1f72